src/patient-registration/patient-registration.component.tsx

```diff
@@ -385,37 +341,16 @@ export const PatientRegistration: React.FC = () => {
   const onFormSubmit = async (values: FormValues) => {
     const abortController = new AbortController();
     const relationships = values.relationships;
-    const identifiers = [];
-
-    for (const type of identifierTypes) {
-      const idValue = values[type.fieldName];
-
-      if (idValue) {
-        identifiers.push({
-          uuid: patientUuidMap[type.fieldName] ? patientUuidMap[type.fieldName].uuid : undefined,
-          identifier: idValue,
-          identifierType: type.uuid,
-          location: location,
-          preferred: type.isPrimary,
-        });
-      } else if (type.autoGenerationSource) {
-        const response = await generateIdentifier(type.autoGenerationSource.uuid, abortController);
-
-        identifiers.push({
```

Comment:
i'm curious why the uuid in this case is non existent here but maybe im assuming this is the case for a POST call and an UPDATE call.